neqo-transport/src/recovery/mod.rs

```diff
@@ -658,41 +642,40 @@ impl LossRecovery {
         // We need to ensure that we have sent any PTO probes before they are removed
         // as we rely on the count of in-flight packets to determine whether to send
         // another probe.  Removing them too soon would result in not sending on PTO.
-        let loss_delay = primary_path.borrow().rtt().loss_delay();
         let cleanup_delay = self.pto_period(primary_path.borrow().rtt());
-        let mut lost = Vec::new();
-        self.spaces.get_mut(pn_space).unwrap().detect_lost_packets(
-            now,
-            loss_delay,
-            cleanup_delay,
-            &mut lost,
-        );
-        self.stats.borrow_mut().lost += lost.len();
+        if let Some(sp) = self.spaces.get_mut(pn_space) {
```

Comment:
This is another great candidate for `let Some(foo) = expr else { return ...; };`